bindings/rust/s2n-tls-hyper/Cargo.toml

```diff
@@ -16,9 +16,14 @@ default = []
 s2n-tls = { version = "=0.3.7", path = "../s2n-tls" }
 s2n-tls-tokio = { version = "=0.3.7", path = "../s2n-tls-tokio" }
 hyper = { version = "1" }
-hyper-util = { version = "0.1", features = ["client-legacy", "tokio", "http1"] }
+hyper-util = { version = "0.1", features = ["client-legacy", "tokio", "http1", "http2"] }
 tower-service = { version = "0.3" }
-http = { version= "1" }
+http = { version = "1" }
+
+# Newer versions require Rust 1.65, see https://github.com/aws/s2n-tls/issues/4242.
+hashbrown = { version = "=0.15.0" }
```

Comment:
Our standard pattern is to have "pins" in the dev-dependencies.

That way any library consumers with less onerous MSRV requirements can still use the newer versions of these.